packages/cms-cli/commands/functions/build.js

```diff
@@ -0,0 +1,73 @@
+const ora = require('ora');
+const {
+  addPortalOptions,
+  addConfigOptions,
+  setLogLevel,
+  getPortalId,
+  addUseEnvironmentOptions,
+} = require('../../lib/commonOpts');
+const { trackCommandUsage } = require('../../lib/usageTracking');
+const { logDebugInfo } = require('../../lib/debugInfo');
+const {
+  loadConfig,
+  validateConfig,
+  checkAndWarnGitInclusion,
+} = require('@hubspot/cms-lib');
+const { logger } = require('@hubspot/cms-lib/logger');
+const { buildPackage } = require('@hubspot/cms-lib/api/functions');
+const { validatePortal } = require('../../lib/validation');
+
+const makeSpinner = (functionPath, portalIdentifier) => {
+  return ora(
+    `Building new bundle for '${functionPath}' on portal '${portalIdentifier}'.\n`
+  );
+};
+
+const loadAndValidateOptions = async options => {
+  setLogLevel(options);
+  logDebugInfo(options);
+  const { config: configPath } = options;
+  loadConfig(configPath, options);
+  checkAndWarnGitInclusion();
+
+  if (!(validateConfig() && (await validatePortal(options)))) {
+    process.exit(1);
+  }
+};
+
+exports.command = 'build <path>';
+exports.describe =
+  'builds a new dependency bundle for the specified function folder';
+
+exports.handler = async options => {
+  loadAndValidateOptions(options);
+
+  const { path: functionPath } = options;
+  const portalId = getPortalId(options);
+  const spinner = makeSpinner(functionPath, portalId);
+
+  trackCommandUsage('functions-build', { functionPath }, portalId);
+
+  logger.debug(`Starting build for function(s) with path: ${functionPath}`);
+
+  spinner.start();
+  await buildPackage(portalId, `${functionPath}/package.json`);
```

Comment:
Do we need to handle the scenario where a package.json doesn't exist?  Since legacy functions won't have one

Reply:
Also just an idea, but since a CLI user may not know off of the top of their head the remote path to the function they want to build, making an interactive selector would be sweet.  It could leverage the `hs functions list` to get the remote paths

Reply:
> Do we need to handle the scenario where a package.json doesn't exist? Since legacy functions won't have one

Added.

> It could leverage the hs functions list to get the remote paths

Love this idea! The endpoint does not currently support passing an endpoint ATM, but we should add that capability so we can then use the list to create a interactive selector @bkrainer